cpu/nrf5x_common/periph/pm.c

```diff
@@ -23,8 +23,14 @@
 
 #include "cpu.h"
 
+#ifdef NRF_POWER_S
+#define NRF_POWER NRF_POWER_S
+#endif
+
+/* TODO: implement proper pm_off for nRF9160 */
```

Comment:
Is this even compiled if `periph_pm` is not advertised as present?

Reply:
I added it because Murdock was complaining.